environment.yml

```diff
@@ -0,0 +1,20 @@
+name: brain-score.web
+channels:
+- defaults
+- anaconda
+- pytorch
+dependencies:
+  - python>=3.6
+  - pip
+  - django
+  - numpy
+  - psycopg2
+  - nodejs
+  - boto3
+  - pandas
+  - pip:
+    - django_compressor
+    - colour
+    - tqdm
+    - awsebcli
+    - requests
```

Comment:
do we want to use a `setup.py` *and* an `environment.yml`?

Reply:
I think that's pretty standard.  

Some reputable-looking repos with both:  
- https://github.com/rosalindfranklininstitute/cuda-slic
- https://github.com/NREL/powerscenarios
- https://github.com/QuantStack/ipycytoscape
- https://github.com/flatironinstitute/neuropixels-data-sep-2020